roles/public_facing/files/download.ceph.com/rrsync

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/perl
```

Comment:
It still bugs me that we're including a custom one rather than the one that comes with rsync.  If we're really gonna, we ought to insert a comment saying that we are, and why, so that someone looking doesn't get confused in a year